### PR TITLE
Feature/styling post details page

### PR DESCRIPTION
**Pull request by niklasrammelkamp**
Hi, here's my pull request to the following User Story:
[Styling: Post Details Page](https://github.com/niklasrammelkamp/capstone-project/issues/38#issue-1579694028)

### CODE REVIEW

File: components/CommentList/index.js [hunk 1]
```diff
@@ -1,52 +1,84 @@
...
-                >
-                  <SVGIcon variant="bin" width="13" />
-                </button>
-              )}
+              <StyledCommentContent>{comment.content}</StyledCommentContent>
             </StyledComment>
           );
-        })
+        })}
+      {comments.length > 5 ? (
```
Comment (doemser) [inline components/CommentList/index.js:53]:
```suggestion
      {comments.length > 5 && (
```

and then leave the empty fragments out.

**Review by doemser — CHANGES_REQUESTED** [review]
Looks pretty good, just found some minor things.

File: pages/_app.js [hunk 2]
```diff
@@ -19,7 +19,7 @@ export default function App({
       >
         <GlobalStyle />
         <Head>
-          <title>Foto App</title>
+          <title>Click</title>
```
[the remark this answers is not in the document]
Comment (doemser) [reply pages/_app.js:22]: Oh its the app name :D nevermind

File: components/CommentForm/index.js [hunk 3]
```diff
@@ -1,29 +1,39 @@
...
+import Button from "../Button";
 
 export default function CommentForm({ onAddComment }) {
   function handleSubmit(event) {
     event.preventDefault();
     onAddComment(event.target.elements.comment.value);
 
     event.target.reset();
-    event.target.elements.comment.style.height = "28px";
+    event.target.elements.comment.style.height = "4rem";
```
[the remark this answers is not in the document]
Comment (niklasrammelkamp) [reply components/CommentForm/index.js:12]: Don't need it anymore. I delete it.

File: components/Textarea/index.js [hunk 4]
```diff
@@ -7,31 +8,70 @@ export default function Textarea({
...
         maxLength={maxLength}
         defaultValue={defaultValue}
+        onFocus={() => setInputFocus(true)}
+        onBlur={(event) => {
+          if (event.target.value) {
+            setInputFocus(true);
+          } else {
+            setInputFocus(false);
+          }
+        }}
```
[the remark this answers is not in the document]
Comment (niklasrammelkamp) [reply components/Textarea/index.js:38]: @doemser 
No, I don't think 'cause the label needs to know if the textarea is in focus or if something is left in the textarea by leaving the focus.

**Pull request merged**